content/en/docs/contribute/participate/pr-wranglers.md

```diff
@@ -87,3 +87,24 @@ To close a pull request, leave a `/close` comment on the PR.
 The [`fejta-bot`](https://github.com/fejta-bot) bot marks issues as stale after 90 days of inactivity. After 30 more days it marks issues as rotten and closes them.  PR wranglers should close issues after 14-30 days of inactivity.
 
 {{< /note >}}
+
+# PR Wrangler Shadow Program
+
+In late 2021, SIG Docs introduced the PR Wrangler Shadow Program. The program was introduced to help new contributors to understand the PR wrangling process and the PR worklflow.
+
+### Become a Shadow
+
+- If you are interested in participating in the PR wrangler shadow mentoring visit the [PR Wranglers Wiki page](https://github.com/kubernetes/website/wiki/PR-Wranglers) to see the wrangling schedule for the year.
+
+- You can reach out to Brad Topol (`@bradtopol` on Slack) and register you interest in becoming a shadow for the PR Wrangler role for a particular week.
+
+### Learning Exceptations
+
+While shadowing a PR wrangler for a particular week, participants will learn:
+  
+  - The wrangling process which starts when a PR is opened and ends when a PR is merged.
+  - Duties of a PR Wrangler.
+  - Useful Kubernetes Prow commands.
+  - The considerations while applying the `/lgtm` and `/approve` labels.
+  - How to interact with people while wrangling a PR.
+  - How to tackle different types of PRs ranging from inactive PRs, PRs needing CLA signing and PRs needing technical reviews for other SIGs.
```

Comment:
I'm not sure if this section is needed since this page is about PR Wranglers
This PR adds a small section on how to shadow a PR Wrangler hence learn what the PR Wrangler duties are which have already been stated under the **Duties** section

Reply:
I think we should keep this section because for newcomers it helps them to get a clear idea for what they are going to learn after shadowing (cause I believe every wrangler imparts some knowledge which the _duties_ section does not cover), which will  encourage more people to be a part of the shadow program. Thoughts? 
I am open to all your feedback and will do the needful changes if necessary. Thanks.